xml/file_management.xml

```diff
@@ -131,20 +131,20 @@
 
    <para>
     The <command>umask</command> (user file-creation mode mask) command is a
-    shell built-in command which determines the default file permissions for
+    shell built-in command that determines the default file permissions for
     newly created files and directories. This can be overwritten by system calls but many
     programs and utilities use <command>umask</command>. By default,
     <command>umask</command> is set to <literal>022</literal>. You can modify
     this globally by changing the value in <filename>/etc/profile</filename>
-    or for each user in the startup files of the shell.
+    or for each user in the start-up files of the shell.
    </para>
 
    <para>
     To determine the active umask, use the <command>umask</command>
     command:
    </para>
 
-<screen>&prompt.user;umask
+<screen>&prompt.user;umask 
```

Comment:
I see no need for the trailing space but as it is invisible I'll leave it just in case.